### PR TITLE
Updated device definition for Meazon meters and associated converter compatible with current Meazon firmware

### PR DESCRIPTION
**Pull request by ccorderod**
This is an updated device definition for Meazon meters and associated converter compatible with current Meazon firmware release for those devices.

### CODE REVIEW

**Comment by ccorderod** [comment]
As requested in #1699 . Txs.

File: devices.js [hunk 1]
```diff
@@ -11129,14 +11135,19 @@ const devices = [
...
+                minimumReportInterval: 1,
+                maximumReportInterval: 5,
+                reportableChange: 1,
+            }], options);
             await endpoint.configureReporting('seMetering', [{
-                attribute: {ID: 0x2000, type: 0x29},
+                attribute: {ID: 0x3000, type: 0x23},
                 minimumReportInterval: 1,
-                maximumReportInterval: repInterval.MINUTES_5,
+                maximumReportInterval: 5,
```
Comment (Koenkk) [inline devices.js:11150]: 5 seconds seems very low for a max report interval. This means it will report every 5 seconds even when no change was observed.

**Comment by Koenkk** [comment]
Can you post the herdsman debug logging of this issue:

```
error 2020-10-21 15:21:27: Failed to setup reporting for '0x00124b0002cc3cba' - TypeError: Cannot read property 'groupID' of undefined at Endpoint.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:281:103) at Generator.next (<anonymous>) at /opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:27:71 at new Promise (<anonymous>) at __awaiter (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:23:12) at Endpoint.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/model/endpoint.js:275:16) at Report.setupReporting (/opt/zigbee2mqtt/lib/extension/report.js:150:38) at Report.onZigbeeEvent (/opt/zigbee2mqtt/lib/extension/report.js:225:24) at Controller.callExtensionMethod (/opt/zigbee2mqtt/lib/controller.js:372:44)
```

To enable herdsman debug logging, see https://www.zigbee2mqtt.io/information/debug.html#zigbee-herdsman-debug-logging

**Comment by ccorderod** [comment]
Morning @Koenkk 
Here is the herdsman debug log requested: https://pastebin.com/749AzEUG
Stay safe. Carlos

**Comment by Koenkk** [comment]
Thanks, can you share your `data/database.db`? (and what coordinator are you using?)

**Comment by ccorderod** [comment]
Here you have @Koenkk: https://pastebin.com/TYifmkqP
I use CC1352P_2, running latest beta fw (20201113) and z2m 1.15.0-dev.
Thank you.

**Comment by Koenkk** [comment]
Does the issue stop when setting `report` to `false` in your configuration.yaml?

**Comment by ccorderod** [comment]
Hi @Koenkk . Yes, the issue stop when I set report to false in configuration.yaml.

But when I start Z2M i get a few of those:

`Zigbee2MQTT:error 2020-12-09 17:44:13: Failed to disable reporting for '0x90fd9ffffed58a2d' - TypeError: Unbind 0x90fd9ffffed58a2d/1 genOnOff from 'undefined' failed (Cannot read property 'toString' of undefined)
    at ZStackAdapter.toAddressString (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:855:28)
    at ZStackAdapter.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:574:38)
    at Generator.next (<anonymous>)
    at /opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:27:71
    at new Promise (<anonymous>)
    at __awaiter (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:23:12)
    at Object.func (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:564:45)
    at Queue.<anonymous> (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/utils/queue.js:29:46)
    at Generator.next (<anonymous>)
    at /opt/zigbee2mqtt/node_modules/zigbee-herdsman/dist/utils/queue.js:8:71`

Those are not Meazon meters but all ikea bulbs.

FYI, I just updated the 1352P_2 coordinator to latest dev firmare (20201113) and upgraded z2m to 1.16.2-dev (commit 7ca8535).

Should I keep the report to false from now on?

Thank you. Stay safe.

**Comment by Koenkk** [comment]
Still have no clue why it's not working. Can you update `/opt/zigbee2mqtt/lib/extension/report.js` with https://gist.github.com/Koenkk/ed5b2de2fb646e8971aa87e87f7d6364 and provide me the logging from starting z2m till these errors?

**Comment by ccorderod** [comment]
un-stale

**Comment by Koenkk** [comment]
@ccorderod please check https://github.com/Koenkk/zigbee-herdsman-converters/pull/1700#issuecomment-742683644

**Comment by ccorderod** [comment]
@Koenkk Shame on me :(. I miss that one. Apologies. Will do.
Happy 2021 (from Madrid, under the snow :-))

**Comment by ccorderod** [comment]
Morning.
Better late than never. Here you go: https://pastebin.com/BV3fwY6E
Have a good day.

**Comment by Koenkk** [comment]
With the next release the report feature will become deprecated so I don't think we should invest time in it. Can you set `report: false` in your configuration.yaml` and remove  `"reporting":1` from your databse.db for every device?

e.g.
`"meta":{"reporting":1,"configured":24},"lastSeen":1606329893652} ` should become `"meta":{"configured":24},"lastSeen":1606329893652}`.

Do you get any errors after that?

**Comment by ccorderod** [comment]
All clean Koenkk :-). No more errors or even warnings on the log.
BTW, I did upgrade to latest dev 1.17.0-dev and upgraded the CC1352_P2_launchpad to latest dev firmware.
The current PR does not make sense anymore. May you please revoke it or should I do that myself?
I will PR a new one with new Meazon device definition and associated converter compatible with current Z2M version.
Thank you. Stay safe.

**Pull request closed without merging**